File: Java/jobexpenses.java
/*
 *  Problem: "Job Expenses"
 */

import java.util.Scanner;

public class jobexpenses {
	
	/*-----mySolution-----*/
	public static int mySolution(Scanner sc) {
		int expenses = 0;
		int reciepts = sc.nextInt();
		
		for (int i = 0; i < reciepts; i++) {
			int item = sc.nextInt();
			if (item < 0)
				expenses += item;
		}
		
		expenses = Math.abs(expenses);
		return expenses;
	}
	
	/*-----main-----*/
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int expenses = mySolution(sc);
		System.out.println(expenses);
	}
	
}

File: Ruby/jobexpenses.rb
=begin
 *
 *	Problem:		Job Expenses
 *	Problem ID:	jobexpenses
 *	url:				https://open.kattis.com/problems/jobexpenses
 *	Solution:		Ruby
 *
=end

class Solver

	def input_to_int(input)
		input_array = input.chomp.split(" ")
		input_int = input_array.map(&:to_i)
		return input_int
	end
	
	def solve
		number_receipts = input_to_int(gets)
		receipts = input_to_int(gets)
		expenses = 0
		receipts.each do |receipt|
			if (receipt < 0)
				expenses += receipt.abs
			end
		end
		puts(expenses)
	end
	
end

solver = Solver.new
solver.solve
